src/sass/style.scss

```diff
@@ -64,3 +64,31 @@ body {
 .grav-c-two-columns-text {
   margin: 2rem auto 0em auto;
 }
+
+// Temporary overrides - About page partner logos
+.grav-c-list-inline-row {
+  margin-bottom: $grav-sp-m;
+  justify-content: center;
+}
+
+.grav-c-list-inline-row>li {
+  flex: auto 0 0;
+  margin-right: $grav-sp-m;
+  margin-left: $grav-sp-m;
+}
+
+.grav-c-list-inline-row a {
+  height: auto;
```

Comment:
You can remove this line. Height defaults to `auto` anyway, so it's redundant.

Reply:
Done too